doc/Publishing.md

```diff
@@ -2,10 +2,19 @@
 
 In order to release one should:
 
+- pre-publishing steps
 - login to the npm registry
 - publish packages
 - tag the published version
 
+## Pre-Publishing Steps
+Before publishing it's important to make sure that theia-apps builds against 'next'. Else we will have problems with "latest" after publishing
```

Comment:
Please make sure we consistently use 'next' and "next". It is up to you which one you use, just pick one.

Reply:
or `next` :)